pkgs/development/web/insomnia/default.nix

```diff
@@ -1,9 +1,46 @@
-{ stdenv, makeWrapper, fetchurl, dpkg, alsaLib, atk, cairo, cups, dbus, expat
-, fontconfig, freetype, gdk-pixbuf, glib, gnome2, pango, nspr, nss, gtk3, gtk2
-, at-spi2-atk, gsettings-desktop-schemas, gobject-introspection, wrapGAppsHook
-, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
-, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, nghttp2
-, libudev0-shim, glibc, curl, openssl, autoPatchelfHook }:
+{ stdenv
```

Comment:
Any changes here? I don't feel like comparing 42 dependencies to find the one that changed.

Reply:
No changes, I've just used [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt).